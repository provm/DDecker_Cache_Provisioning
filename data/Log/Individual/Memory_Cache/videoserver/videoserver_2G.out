========== Container 4 Memory STATS ==============
cache 23891968
rss 7503872
rss_huge 0
mapped_file 8032256
writeback 0
pgpgin 77471
pgpgout 69806
pgfault 172433
pgmajfault 196
inactive_anon 53248
active_anon 7536640
inactive_file 10842112
active_file 12963840
unevictable 0
hierarchical_memory_limit 999997440
total_cache 23891968
total_rss 7503872
total_rss_huge 0
total_mapped_file 8032256
total_writeback 0
total_pgpgin 77471
total_pgpgout 69806
total_pgfault 172433
total_pgmajfault 196
total_inactive_anon 53248
total_active_anon 7536640
total_inactive_file 10842112
total_active_file 12963840
total_unevictable 0
======== END Container STATS ======
WARNING: Could not open /proc/sys/kernel/shmmax file!
It means that you probably ran Filebench not as a root. Filebench will not increase shared
region limits in this case, which can lead to the failures on certain workloads.
Filebench Version 1.4.9.1
 1851: 0.000: Allocated 170MB of shared memory
 1851: 0.014: Eventgen rate taken from variable
 1851: 0.015: Video Server Version 3.0 personality successfully loaded
 1851: 0.015: Creating/pre-allocating files and filesets
 1851: 0.051: Fileset passivevids: 10 files, 0 leafdirs, avg dir width = 20, avg dir depth = 0.8, 5699.813MB
 1851: 0.051: Re-using fileset passivevids.
 1851: 0.051: Creating fileset passivevids...
 1851: 0.051: Preallocated 6 of 10 of fileset passivevids in 1 seconds
 1851: 0.051: Fileset activevids: 5 files, 0 leafdirs, avg dir width = 4, avg dir depth = 1.2, 7474.542MB
 1851: 0.051: Re-using fileset activevids.
 1851: 0.051: Creating fileset activevids...
 1851: 0.086: Preallocated 5 of 5 of fileset activevids in 1 seconds
 1851: 0.086: waiting for fileset pre-allocation to finish
 1880: 21.659: Starting 1 vidreaders instances
 1880: 21.754: Starting 1 vidwriter instances
 1882: 21.775: Starting 1 vidwriter threads
 1881: 21.798: Starting 16 vidreaders threads
 1851: 22.820: Running...
 1851: 1823.521: Run took 1800 seconds...
 1851: 1827.156: Per-Operation Breakdown
vidreader            21390008ops    11856ops/s 2961.4mb/s      1.2ms/op     2688us/op-cpu [0ms - 2512ms]
replaceinterval      57ops        0ops/s   0.0mb/s  20000.3ms/op        0us/op-cpu [20000ms - 20002ms]
wrtclose             58ops        0ops/s   0.0mb/s      0.0ms/op        0us/op-cpu [0ms - 0ms]
newvid               58ops        0ops/s  18.4mb/s  11169.8ms/op   945517us/op-cpu [89ms - 32923ms]
wrtopen              58ops        0ops/s   0.0mb/s      1.7ms/op      345us/op-cpu [0ms - 86ms]
vidremover           58ops        0ops/s   0.0mb/s    164.0ms/op     4483us/op-cpu [0ms - 2240ms]
 1851: 1827.156: IO Summary: 21390240 ops, 11856.605 ops/s, (11856/0 r/w), 2979.8mb/s,    177us cpu/op,   1.2ms latency
 1851: 1827.156: Shutting down processes
========== Container 4 Memory STATS AT END ==============
cache 831492096
rss 1134592
rss_huge 0
mapped_file 2093056
writeback 0
pgpgin 66240132
pgpgout 66036854
pgfault 43046585
pgmajfault 4019
inactive_anon 995328
active_anon 143360
inactive_file 300068864
active_file 531390464
unevictable 0
hierarchical_memory_limit 999997440
total_cache 831492096
total_rss 1134592
total_rss_huge 0
total_mapped_file 2093056
total_writeback 0
total_pgpgin 66240132
total_pgpgout 66036854
total_pgfault 43046585
total_pgmajfault 4019
total_inactive_anon 995328
total_active_anon 143360
total_inactive_file 300068864
total_active_file 531390464
total_unevictable 0
======== END Container STATS ======
