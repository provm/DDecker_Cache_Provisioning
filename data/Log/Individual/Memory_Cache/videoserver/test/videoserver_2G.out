========== Container filebench Memory STATS ==============
cache 23916544
rss 7733248
rss_huge 0
mapped_file 8032256
writeback 0
swap 0
pgpgin 78153
pgpgout 70426
pgfault 174330
pgmajfault 195
inactive_anon 49152
active_anon 7712768
inactive_file 10936320
active_file 12894208
unevictable 0
hierarchical_memory_limit 999997440
hierarchical_memsw_limit 9223372036854771712
total_cache 23916544
total_rss 7733248
total_rss_huge 0
total_mapped_file 8032256
total_writeback 0
total_swap 0
total_pgpgin 78153
total_pgpgout 70426
total_pgfault 174330
total_pgmajfault 195
total_inactive_anon 49152
total_active_anon 7712768
total_inactive_file 10936320
total_active_file 12894208
total_unevictable 0
======== END Container STATS ======
WARNING: Could not open /proc/sys/kernel/shmmax file!
It means that you probably ran Filebench not as a root. Filebench will not increase shared
region limits in this case, which can lead to the failures on certain workloads.
Filebench Version 1.4.9.1
 1857: 0.000: Allocated 170MB of shared memory
 1857: 0.011: Eventgen rate taken from variable
 1857: 0.011: Video Server Version 3.0 personality successfully loaded
 1857: 0.011: Creating/pre-allocating files and filesets
 1857: 0.039: Fileset passivevids: 10 files, 0 leafdirs, avg dir width = 20, avg dir depth = 0.8, 5699.813MB
 1857: 0.039: Re-using fileset passivevids.
 1857: 0.039: Creating fileset passivevids...
 1857: 0.039: Preallocated 6 of 10 of fileset passivevids in 1 seconds
 1857: 0.039: Fileset activevids: 5 files, 0 leafdirs, avg dir width = 4, avg dir depth = 1.2, 7474.542MB
 1857: 0.039: Re-using fileset activevids.
 1857: 0.039: Creating fileset activevids...
 1857: 0.052: Preallocated 5 of 5 of fileset activevids in 1 seconds
 1857: 0.052: waiting for fileset pre-allocation to finish
 1881: 15.382: Starting 1 vidreaders instances
 1881: 15.394: Starting 1 vidwriter instances
 1883: 15.427: Starting 1 vidwriter threads
 1882: 15.442: Starting 16 vidreaders threads
 1857: 16.443: Running...
 1857: 1816.643: Run took 1800 seconds...
 1857: 1817.027: Per-Operation Breakdown
vidreader            18625656ops    10345ops/s 2584.0mb/s      1.3ms/op     1308us/op-cpu [0ms - 1251ms]
replaceinterval      71ops        0ops/s   0.0mb/s  20000.1ms/op        0us/op-cpu [20000ms - 20000ms]
wrtclose             72ops        0ops/s   0.0mb/s      0.0ms/op        0us/op-cpu [0ms - 0ms]
newvid               72ops        0ops/s  22.5mb/s   5130.0ms/op   740278us/op-cpu [151ms - 20999ms]
wrtopen              72ops        0ops/s   0.0mb/s      1.8ms/op      417us/op-cpu [0ms - 79ms]
vidremover           72ops        0ops/s   0.0mb/s     50.2ms/op     5556us/op-cpu [0ms - 713ms]
 1857: 1817.027: IO Summary: 18625944 ops, 10345.567 ops/s, (10345/0 r/w), 2606.5mb/s,     92us cpu/op,   1.3ms latency
 1857: 1817.027: Shutting down processes
========== Container 1 Memory STATS AT END ==============
cache 831086592
rss 995328
rss_huge 0
mapped_file 2093056
writeback 0
swap 5758976
pgpgin 61801458
pgpgout 61598313
pgfault 37523874
pgmajfault 2670
inactive_anon 815104
active_anon 180224
inactive_file 208146432
active_file 622915584
unevictable 0
hierarchical_memory_limit 999997440
hierarchical_memsw_limit 9223372036854771712
total_cache 831086592
total_rss 995328
total_rss_huge 0
total_mapped_file 2093056
total_writeback 0
total_swap 5758976
total_pgpgin 61801458
total_pgpgout 61598313
total_pgfault 37523874
total_pgmajfault 2670
total_inactive_anon 815104
total_active_anon 180224
total_inactive_file 208146432
total_active_file 622915584
total_unevictable 0
======== END Container STATS ======
